.github/workflows/windows.yml

```diff
@@ -17,59 +17,43 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - name: VS2019WChar
-            vmimage: windows-2019
-            mpctype: vs2019
+          - name: VS2022WChar
+            vmimage: windows-2022
+            mpctype: vs2022
             BuildPlatform: x64
             BuildConfiguration: Debug
             vcpkgarch: x64-windows
             vcpkglibdir: debug/lib
             vcpkgpackages: '"openssl", {"name": "xerces-c", "features": ["xmlch-wchar"]}, "zlib"'
             OptionalFeatures: uses_wchar=1
-          - name: VS2019Debug64
-            vmimage: windows-2019
-            mpctype: vs2019
-            BuildPlatform: x64
-            BuildConfiguration: Debug
-            vcpkgarch: x64-windows
-            vcpkglibdir: debug/lib
-            vcpkgpackages: '"openssl", "xerces-c", "zlib"'
-          - name: VS2019Release64
-            vmimage: windows-2019
-            mpctype: vs2019
-            BuildPlatform: x64
-            BuildConfiguration: Release
-            vcpkgarch: x64-windows
-            vcpkglibdir: lib
-            vcpkgpackages: '"openssl", "xerces-c", "zlib"'
-          - name: VS2019Debug32
-            vmimage: windows-2019
-            mpctype: vs2019
+          - name: VS2022Debug32
+            vmimage: windows-2022
+            mpctype: vs2022
             BuildPlatform: Win32
             BuildConfiguration: Debug
             vcpkgarch: x86-windows
             vcpkglibdir: debug/lib
             vcpkgpackages: '"openssl", "xerces-c", "zlib"'
-          - name: VS2019Release32
-            vmimage: windows-2019
-            mpctype: vs2019
+          - name: VS2022Release32
+            vmimage: windows-2022
+            mpctype: vs2022
             BuildPlatform: Win32
             BuildConfiguration: Release
             vcpkgarch: x86-windows
             vcpkglibdir: lib
             vcpkgpackages: '"openssl", "xerces-c", "zlib"'
-          - name: VS2019Debug64Cxx17
-            vmimage: windows-2019
-            mpctype: vs2019
+          - name: VS2022Debug64Cxx17
```

Comment:
Cop17 is the default so the additional mpc args could be removed

Reply:
Projects in VS2022 default to C++14.

Reply:
Mpc does set cpp17 as default, that is the minimum requirement for ace 800 and newer

Reply:
I'd rather have it in the overall project generation steps.  There could be projects that don't inherit `acedefaults.mpb`.

Reply:
OK, thanks for linking to that.  It's a different case than what I was thinking of.
I'll just change it here.

I did a test to see which setting takes effect when both `-value_template` and `specific` are used.  Unfortunately it's `-value_template` (there's probably a good reason for that).